pkg/integrations/prometheus/promql/metricsconsts.go

```diff
@@ -210,9 +210,65 @@ const metricFunctions = `[
 		"isTimeRangeFunc": true
 	},
 	{
-		"fn": "quantile_over_time", 
-		"name": "Quantile Over Time", 
-		"desc": "The φ-quantile (0 ≤ φ ≤ 1) of the values in the specified interval.", 
+		"fn": "hour",
+		"name": "Hour",
+		"desc": "Extracts the hour of a timestamp, represented as the number of hours since midnight.",
+		"eg": "hour(timestamp(system.disk.used))",
+		"isTimeRangeFunc": false
+	},
+	{
+		"fn": "minute",
+		"name": "Minute",
+		"desc": "Extracts the minute of a timestamp, represented as the number of minutes since the last hour.",
+		"eg": "minute(timestamp(system.disk.used))",
+		"isTimeRangeFunc": false
+	},
+	{
+		"fn": "month",
+		"name": "Month",
+		"desc": "Extracts the month of a timestamp, represented as a number from 1 (January) to 12 (December).",
+		"eg": "month(timestamp(system.disk.used))",
+		"isTimeRangeFunc": false
+	},
+	{
+		"fn": "year",
+		"name": "Year",
+		"desc": "Extracts the year of a timestamp.",
+		"eg": "year(timestamp(system.disk.used))",
+		"isTimeRangeFunc": false
+	},
+	{
+		"fn": "day_of_month",
+		"name": "Day of Month",
+		"desc": "Extracts the day of the month from a timestamp, represented as a number from 1 to 31.",
+		"eg": "day_of_month(timestamp(system.disk.used))",
+		"isTimeRangeFunc": false
+	},
+	{
+		"fn": "day_of_week",
+		"name": "Day of Week",
+		"desc": "Extracts the day of the week from a timestamp, represented as a number from 1 (Monday) to 7 (Sunday).",
+		"eg": "day_of_week(timestamp(system.disk.used))",
+		"isTimeRangeFunc": false
+	},
```

Comment:
This discription doesn't match promQL: https://prometheus.io/docs/prometheus/latest/querying/functions/#day_of_week
It should be 0 to 6 inclusive, with 0=Sunday, 1=Monday, ...

Reply:
I just updated it